### ISSUE TITLE
Admin Accessibility:  Page Blocks - Use Grouping and Group Labels

### ISSUE THREAD

**Issue by carlj11** [omeka/omeka-s#2172]
Issue reported by our Institute accessibility test team.  Each block should have a group label. This provides context as the screen reader user tabs through the interface. Right now, all the user hears is the label of the thing they land on. For instance, most blocks have a “collapse” button, or a “remove value” button. As you tab through and land on “remove value”, you then need to back up and figure out what block / group your actually removing a value from. This is inefficient and error prone.

use fieldset and legend elements to create element groups (standard HTML; no aria required)
or use aria-labelledby (requires aria, lots of element IDs)
